platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/expressions/Expression.java

```diff
@@ -3224,6 +3224,11 @@ public static Expression collator(Expression caseSensitive, Expression diacritic
     return new Expression("collator", new ExpressionMap(map));
   }
 
+  public static Expression format(Expression input) {
```

Comment:
If the plan is to introduce the full `format` expression implementation in a follow-up PR, I'd rather skip the public API + tests part here entirely.

Reply:
Hmm, like instead of modifying the test to use this, just omit the test entirely for the `text-field` case?

Reply:
Exactly, and remove the public API entry from `Expression` class.